### ISSUE TITLE
ACC-264.1(Level A) _Headers_   extra info is read out twice

### ISSUE THREAD

**Issue by Priya1Pavan** [kyma-project/busola#3274]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	Headers
Affected Check Point:	ACC-264.1
Short text:	ACC-264.1(Level A) _Headers_   extra info is read out twice

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. 
•	Select +Upload YAML.
•	Copy the following code and paste it into the editor: 

apiVersion: v1
kind: Namespace
metadata:
  name: nginx-ns
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: nginx-deployment
  namespace: nginx-ns
  labels:
    app: nginx
spec:
  replicas: 2
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - name: nginx
        image: nginx:1.14.2
        ports:
        - containerPort: 80
•	Select Upload
•	Select Close
•	Explore the Monitoring and Health section of the Cluster Details page.
•	In the Events section, select All Messages.
•	Select the sorting icon.
•	In the Sort By section, select Last seen.
•	Select OK.

3. Issue Type: Manual
Observed Behavior:
On accessing the Group headers ‘Sort Order’ and ‘Sort By’ , extra info is read out twice as “List” which is incorrect UI type.
Screenshot:
 
Screenshot 2:
 
Expected Behavior:

Only relevant information about the UI should be communicated to the user. Additional information should be avoided.

[ACC-264.1(Level A) _Table_ Extra announcement.docx](https://github.com/user-attachments/files/16688214/ACC-264.1.Level.A._Table_.Extra.announcement.docx)
[Issue processing-09.2021 (2).docx](https://github.com/user-attachments/files/16688215/Issue.processing-09.2021.2.docx)

**Comment by akucharska** [comment]
Fixed in https://github.com/kyma-project/busola/pull/4062
